packages/cli/src/deploy/getResourceIds.ts

```diff
@@ -15,15 +16,30 @@ export async function getResourceIds({
   // TODO: PR to viem's getLogs to accept topics array so we can filter on all store events and quickly recreate this table's current state
 
   debug("looking up resource IDs for", worldDeploy.address);
-  const logs = await getLogs(client, {
-    strict: true,
-    address: worldDeploy.address,
-    fromBlock: worldDeploy.deployBlock,
-    toBlock: worldDeploy.stateBlock,
-    event: parseAbiItem(storeSpliceStaticDataEvent),
-    args: { tableId: storeTables.store_ResourceIds.tableId },
-  });
+  const logs = await pRetry(
+    () =>
+      getLogs(client, {
+        strict: true,
+        address: worldDeploy.address,
+        fromBlock: worldDeploy.deployBlock,
+        toBlock: worldDeploy.stateBlock,
+        event: parseAbiItem(storeSpliceStaticDataEvent),
+        args: { tableId: storeTables.store_ResourceIds.tableId },
+      }),
+    {
+      retries: 3,
+      onFailedAttempt: async (error) => {
+        const shouldRetry =
+          error instanceof HttpRequestError &&
+          error.status === 400 &&
+          error.message.includes('"block is out of range"');
```

Comment:
do we need the quotes here?
```suggestion
          error.message.includes('block is out of range');
```

Reply:
oh nope! this seems less brittle also